frontend/src/components/poll-in-progress/poll-progress/poll-progress.tsx

```diff
@@ -14,8 +14,8 @@ const PollProgress = (props: PollProgressProps): ReactElement => {
   const classSize: number = data.classSize;
   const responseCount: number = data.responseCount;
 
-  const updateProgress = (event: any): void => {
-    props.updateProgress(event.target.value);
+  const updateProgress = (event: SyntheticEvent): void => {
+    props.updateProgress((event.target as HTMLInputElement).value);
```

Comment:
See lines 17 and 18 for how to properly type react events and to do `event.target.value`. It would be good to update the various `event: any` lines throughout the pr. Not a big deal to leave though since it's warnings and not errors.